std/http/server_test.ts

```diff
@@ -499,9 +491,8 @@ test({
 test({
   name: "respond error closes connection",
   async fn(): Promise<void> {
-    const port = randomPort();
     const serverRoutine = async (): Promise<void> => {
-      const server = serve(":" + port);
+      const server = serve(":8124");
```

Comment:
Same above

Reply:
Let's revert and do another fix up patch rather than conflate the two. It will be easier to manage the changes.